app/domain/authentication/authn_k8s/authentication_service.rb

```diff
@@ -0,0 +1,91 @@
+# AuthenticationService represents the authenticator itself, which 
+# has a Conjur id that:
+#
+# * Identifies a policy.
+# * Identifies a webservice.
+# * Is a naming prefix to CA cert and key variables.
+# * Is a naming prefix to the application hosts.
```

Comment:
per our slack convo, these comments should be udpated to reflect current content of class.  I wonder if `K8sCert` of `ConjurK8sCert` would be a more accurate name now.